tools/dfx-orbit/src/cli.rs

```diff
@@ -0,0 +1,93 @@
+//! Implementation of the `dfx-orbit` commands.
+mod asset;
+mod station;
+
+pub use crate::cli::asset::AssetUploadRequest;
+use crate::{
+    args::{asset::AssetArgsAction, review::ReviewArgs, DfxOrbitArgs, DfxOrbitSubcommands},
+    dfx_extension_api::OrbitExtensionAgent,
+    DfxOrbit,
+};
+
+/// A command line tool for interacting with Orbit on the Internet Computer.
+pub async fn exec(args: DfxOrbitArgs) -> anyhow::Result<()> {
+    let orbit_agent = OrbitExtensionAgent::new()?;
+
+    // We don't need to instanciate a StationAgent to execute this command directly on the orbit agent
+    if let DfxOrbitSubcommands::Station(station_args) = args.command {
+        station::exec(orbit_agent, station_args)?;
+        return Ok(());
+    };
+
+    let mut dfx_orbit = DfxOrbit::new(orbit_agent).await?;
+
+    match args.command {
+        DfxOrbitSubcommands::Me => {
+            let ans = dfx_orbit.station.me().await?;
+            println!("{}", serde_json::to_string_pretty(&ans)?);
+            Ok(())
+        }
+        DfxOrbitSubcommands::Request(request_args) => {
+            let response = dfx_orbit
+                .station
+                .request(request_args.into_create_request_input(&dfx_orbit)?)
+                .await?;
+            let request_id = &response.request.id;
+            let request_url = dfx_orbit.station.request_url(request_id);
+            println!("Created request: {request_id}");
```

Comment:
nit: wasn't there a logger for this?

Reply:
So the whole output story is still a bit of a mess right now.
Right now, it uses a mix of logger and println.


Going forward, I would like to have the following design:
- **Logger**: Logs *internal* messages. A flag sets, how verbose the logging output will be. Goes to stderr
- **Output**: The output that is the result of the execution. This can be manipulated through flags (i.e. have json output). Goes to stdout

I have not had the time to properly implement that yet, but there is a whole overhaul planned. So for the time, I would leave it as is, if thats ok with you.